### DISCUSSION TITLE
How to reconfigure stderr consolde?

### DISCUSSION THREAD

**Discussion by ssbarnea** [Textualize/rich#848] [Q&A]
I seen that you have a `rich.reconfigure()` method that I used successfully to update the main (stdout) console but apparently this does not work for my second stderr console.

What would be the optimal way to reconfigure both of them?

**Comment by willmcgugan** [comment]
The `reconfigure` function is to reconfigure the global console (used by rich.print and progress etc), as there is no convenient way to do that.

If you need to recofigure other Console instances it would depend on how you access it. You could use the `__dict__` trick, or (better) a property on a singleton object, or even a module level `__getatrr__` (Py37+).

**Comment by willmcgugan** [comment]
@ssbarnea Out of interest, what parameter do you want to reconfigure?

I'm contemplating adding a `Console.reconfigure` method.

Also, bear in mind you can modify many parameters of Console after it has been constructed. If its not private is probably safe to modify.
